specification/marketplacecatalog/data-plane/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,1065 @@
+            "name": "isPreferredSolution",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "boolean",
+            "name": "isAdditionalPurchaseRequired",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "boolean",
+            "name": "isPowerBICertified",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "boolean",
+            "name": "isIndustryCloud",
+            "in": "query",
+            "x-nullable": true
+          },
+          {
+            "type": "array",
+            "name": "operatingSystems",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "appSourceIndustries",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "appSourceCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azureIndustries",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azureCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azurePortalCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "x-ms-app",
+            "x-originalName": "callingApp",
+            "in": "header",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "default": "2022-02-02",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of complete terms based on partial search term",
+            "schema": {
+              "$ref": "#/definitions/AutoCompleteResponseDto"
```

Comment:
Is it possible that the response will be so large (contain so many items) that it will require paging?

Reply:
It is a breaking change to add paging later, so if you think you will ever want it, you should design the operation for paging now -- even if you only currently return one page.

Reply:
as discussed. there is no paging planned in the future. we will only return one single page.